draft-ietf-sframe-enc.md

```diff
@@ -118,6 +115,12 @@ E2EE:
 HBH:
 : Hop By Hop
 
+We use "Selective Forwarding Unit (SFU)" and "media stream" in a less formal sense
+than in {{?RFC7656}}.  An SFU is a selective switching function for media
+payloads, and a media stream a sequence of media payloads, in both cases
```

Comment:
Should SFU and media stream be wrapped in quotations here?

Reply:
No, those are just to mark out the terms at introduction..